src/fragments/lib/in-app-messaging/js/customize.mdx

```diff
@@ -0,0 +1,109 @@
+## Integrate Custom Components
+
+Users may provide their own In-App Messaging UI components to override the default Amplify provided UI components by utilizing the `components` prop of `InAppMessagingProvider`.
```

Comment:
I think the target audience here is the customer/developer and so speaking to them about "users" might be confusing since their users are the end users of their application.
```suggestion
You may provide your own In-App Messaging UI components to override the default Amplify provided UI components by utilizing the `components` prop of `InAppMessagingProvider`.
```